Competition2017/tests/competition/subsystems/drive/commands/DriveToPointUsingHeuristicsCommandTest.java

```diff
@@ -0,0 +1,79 @@
+package competition.subsystems.drive.commands;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import competition.subsystems.drive.DriveTestBase;
+import competition.subsystems.pose.PoseSubsystem;
+import edu.wpi.first.wpilibj.MockTimer;
+
+public class DriveToPointUsingHeuristicsCommandTest extends DriveTestBase {
+
+    DriveToPointUsingHeuristicsCommand command;
+    MockTimer timer;
+    
+    @Before
+    public void setUp() {
+        // TODO Auto-generated method stub
+        super.setUp();
+        command = injector.getInstance(DriveToPointUsingHeuristicsCommand.class);
+        timer = injector.getInstance(MockTimer.class);
+    }
+    
+    @Test
+    public void start() {
```

Comment:
it might be nice to give a little more descriptive test method name about what this scenario is